# Tail Sizing

This section describes tail sizing for the example airplane. In the early stages of design, the tail volume coefficient method is used for initial sizing of tail surfaces. The tail volume coefficient is defined as the tail effectiveness non-dimensionalized using the product of wing area and appropriate length scale. The effectiveness of a tail can be measured using the product of tail area and tail moment arm. The length scale is defined based on whether the coefficient is computed for horizontal or vertical tail. Refer to lecture notes, Raymer section 6.5, Roskam part 2 chapter 8, and Nicolai & Carichner chapter 11 for more details.

## Horizontal tail

The horizontal tail volume coefficient is defined as

$$
    V_h = \frac{S_h L_h}{S \bar{c}},
$$

where $S_h$ and $S$ are the horizontal tail surface and main wing areas, respectively, while $\bar{c}$ refers to the mean aerodynamic chord of the wing. The $L_h$ is the horizontal tail moment arm, measured as the distance between the mean aerodynamic centers of the horizontal tail surface and the wing. The value of the coefficient and tail moment arm are estimated based on the historical data for a similar class of airplanes. For the example airplane, the $V_h$ and $L_h$ are set to 0.8 and 15 ft, respectively, based on Table 8.3(a) (Roskam Part 2).

The planform values for the horizontal tail surface are estimated using the historical data presented in Table 8.13, Roskam Part 2. The aspect ratio is set to 5.5, while taper ratio is 0.75. The quarter chord sweep is set to $10^{\circ}$ and the dihedral angle is set to $5^{\circ}$, while the incidence angle is kept variable for trimming purposes. The NACA 0012 is used as the airfoil for the hortizontal tail surface.

Below code block computes the value of horizontal tail surface planform variables based on the equations outlined in [wing planform section](wing_layout):

In [1]:
# wing variables
S = 134 # sq ft
wing_mac = 4.3 # ft

# variables
aspect_ratio_h = 5.5
taper_ratio_h = 0.75
sweep_qc_h = 10 # deg
V_h = 0.8
l_h = 15 # ft

# area
tail_area_h = V_h * S * wing_mac / l_h

# span
b_h = (aspect_ratio_h*tail_area_h)**0.5

# root chord
c_root_h = 2 * tail_area_h / b_h / (1 + taper_ratio_h) # ft

# tip chord
c_tip_h = taper_ratio_h * c_root_h # ft

# MAC
mac_h = 2 * c_root_h * (1 + taper_ratio_h + taper_ratio_h**2) / 3 / (1 + taper_ratio_h)

# y_mac
y_mac_h = b_h * (1 + 2*taper_ratio_h) / 6 / (1 + taper_ratio_h)

# Print values
print(f"Area: {tail_area_h:.0f} sq. ft")
print(f"Span: {b_h:.0f} ft")
print(f"Root chord: {c_root_h:.1f} ft")
print(f"Tip chord: {c_tip_h:.1f} ft")
print(f"MAC: {mac_h:.1f} ft")
print(f"y_mac: {y_mac_h:.1f} ft")

Area: 31 sq. ft
Span: 13 ft
Root chord: 2.7 ft
Tip chord: 2.0 ft
MAC: 2.4 ft
y_mac: 3.1 ft


## Vertical tail

The vertical tail volume coefficient is defined as

$$
    V_v = \frac{S_v L_v}{S b},
$$

where $S_v$ is the vertical tail surface area, while $b$ refers to the wing span. The $L_v$ is the vertical tail moment arm, measured as the distance between the mean aerodynamic centers of the vertical tail surface and the wing. The value of the coefficient and tail moment arm are estimated based on the historical data for a similar class of airplanes. For the example airplane, the $V_v$ and $L_v$ are set to 0.065 and 17 ft, respectively, based on Table 8.3(b) (Roskam Part 2). Next, the planform values for the vertical tail surface are estimated using the historical data presented in Table 8.13 (Roskam Part 2). The aspect ratio is set to 1.3, while taper ratio is 0.55. The quarter chord sweep is set to $30^{\circ}$ and the NACA 0012 is used as the airfoil.

Below code block computes the value of vertical tail surface planform variables:

In [2]:
# wing variables
S = 134 # sq ft
b = 33 # ft

# variables
aspect_ratio_v = 1.3
taper_ratio_v = 0.55
sweep_qc_v = 30 # deg
V_v = 0.065
l_v = 17 # ft

# area
tail_area_v = V_v * S * b / l_v

# span
b_v = (aspect_ratio_v*tail_area_v)**0.5

# root chord
c_root_v = 2 * tail_area_v / b_v / (1 + taper_ratio_v) # ft

# tip chord
c_tip_v = taper_ratio_v * c_root_v # ft

# MAC
mac_v = 2 * c_root_v * (1 + taper_ratio_v + taper_ratio_v**2) / 3 / (1 + taper_ratio_v)

# y_mac, multiplied by 2
y_mac_v = 2 * b_v * (1 + 2*taper_ratio_v) / 6 / (1 + taper_ratio_v)

# Print values
print(f"Area: {tail_area_v:.0f} sq. ft")
print(f"Span: {b_v:.0f} ft")
print(f"Root chord: {c_root_v:.1f} ft")
print(f"Tip chord: {c_tip_v:.1f} ft")
print(f"MAC: {mac_v:.1f} ft")
print(f"y_mac: {y_mac_v:.1f} ft")

Area: 17 sq. ft
Span: 5 ft
Root chord: 4.7 ft
Tip chord: 2.6 ft
MAC: 3.7 ft
y_mac: 2.1 ft


> __*NOTE*:__ The value of the spanwise location of mac obtained from the equation should be multiplied by 2 for a vertical tail surface

## Summary


Following table summarizes the tail surface sizing for the example airplane:

<div style="width:70%; margin: auto;">

Parameter | Horizontal tail | Vertical tail | Source 
--- | :---: | :---: | :---:
Tail volume coefficient | 0.8 | 0.065 | historical data
Tail moment arm | 15 ft | 17 ft | historical data
Surface area | 31 $\text{ft}^2$ | 17 $\text{ft}^2$ | calculated
Span | 13 ft | 5 ft | calculated
Root chord | 2.7 ft | 4.7 ft | calculated
Tip chord | 2 ft | 2.6 ft | calculated 
Mean aerodynamic chord | 2.4 ft | 3.7 ft | calculated
Spanwise location of mac | 3.1 ft | 2.1 ft | calculated
Quarter chord sweep | $10^{\circ}$ | $30^{\circ}$ | historical data
Incidence | variable | - | -
Dihedral | $5^{\circ}$ | - | historical data
Airfoil | NACA 0012 | NACA 0012 | historical data
</div>

> __*NOTE*:__ Most of these values are based on historical estimates, and hence, should be further refined to meet the control and stability requirements of the airplane, if required.